src/clr-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -645,8 +645,18 @@
         border-radius: 0 0 $clr-global-borderradius $clr-global-borderradius;
 
         .pagination {
-            margin-left: 1.5rem;
-            height: calc(1.5rem - #{2 * $clr-default-borderwidth});
+            display: flex;
+            align-items: center;
```

Comment:
Have we tested it on different browsers?

Reply:
![image](https://user-images.githubusercontent.com/1426805/35058172-1ddd00cc-fb85-11e7-9389-56f5ebe80fdb.png)

Yep!